### ISSUE TITLE
[Arm32][Legacy Backend][R2R] mainv1/2.cmd fail

### ISSUE THREAD

**Issue by jashook** [dotnet/coreclr#13952]
The tests are:

```
readytorun/tests/mainv1/mainv1.cmd
readytorun/tests/mainv2/mainv2.cmd
```

Introduced by #13733

```
10:33:34 TestID: mainv1.cmd_79
10:33:34 Path  : readytorun/tests/mainv1/mainv1.cmd
10:33:34 
10:33:34 Started: Wed Sep 13 10:09:00 2017
10:33:34 Started: Wed Sep 13 10:09:02 2017
10:33:34 Output: TEST_CMD_LINE: readytorun\tests\mainv1> C:\Users\DDJOIN\arm64_ci\tests\tests\readytorun\tests\mainv1\mainv1.cmd 
10:33:34 BEGIN EXECUTION
10:33:34         1 file(s) copied.
10:33:34 Microsoft (R) CoreCLR Native Image Generator - Version 4.5.22220.0
10:33:34 Copyright (c) Microsoft Corporation.  All rights reserved.
10:33:34 Native image test.ni.dll generated successfully.
10:33:34 Microsoft (R) CoreCLR Native Image Generator - Version 4.5.22220.0
10:33:34 Copyright (c) Microsoft Corporation.  All rights reserved.
10:33:34 Native image fieldgetter.ni.dll generated successfully.
10:33:34 Microsoft (R) CoreCLR Native Image Generator - Version 4.5.22220.0
10:33:34 Copyright (c) Microsoft Corporation.  All rights reserved.
10:33:34 Native image mainv1.ni.exe generated successfully.
10:33:34  "C:\Users\DDJOIN\arm64_ci\tests\tests\Tests\Core_Root\corerun.exe" mainv1.exe 
10:33:34 Expected: 100
10:33:34 Actual: 3
10:33:34 END EXECUTION - FAILED
10:33:34 FAILED
```

**Comment by hqueue** [comment]
@sjsinju I know you are now out-of-office. 
Please check this and let us know whether can we close this issue.

**Comment by sjsinju** [comment]
@jashook It has resolved and the tests mainv1/2 was reopened in #13933. If there are not any problems, you can close this issue.

**Comment by nattress** [comment]
@sjsinju Closing this issue since the tests are running and passing after the fix in https://github.com/dotnet/coreclr/pull/13933
